package.json

```
@@ -3,12 +3,13 @@
   "version": "4.4.0",
   "description": "Perform login using oauth2 protocol",
   "main": "index.js",
+  "types": "index.d.ts",
   "scripts": {
     "coverage": "npm run unit -- --cov --coverage-report=html",
     "lint": "standard | snazzy",
     "test": "npm run lint && npm run unit && npm run test:typescript",
     "test:typescript": "tsd",
-    "unit": "tap test.js"
+    "unit": "tap -J test/*.test.js"
```

If you plan to use `*`, please wrap the path with `\"`. Otherwise, it will use the shell glob which may behavior differently depends on system.
```suggestion
    "unit": "tap -J \"test/*.test.js\""
```

---

done ^^